packages/apps/public/locales/en/app-accounts.json

```diff
@@ -213,6 +213,7 @@
   "You can set a custom derivation path for this account using the following syntax \"/<soft-key>//<hard-key>\". The \"/<soft-key>\" and \"//<hard-key>\" may be repeated and mixed`. An optional \"///<password>\" can be used with a mnemonic seed, and may only be specified once.": "You can set a custom derivation path for this account using the following syntax \"/<soft-key>//<hard-key>\". The \"/<soft-key>\" and \"//<hard-key>\" may be repeated and mixed`. An optional \"///<password>\" can be used with a mnemonic seed, and may only be specified once.",
   "You can set a custom derivation path for this account using the following syntax \"/<soft-key>//<hard-key>///<password>\". The \"/<soft-key>\" and \"//<hard-key>\" may be repeated and mixed`. The \"///password\" is optional and should only occur once.": "You can set a custom derivation path for this account using the following syntax \"/<soft-key>//<hard-key>///<password>\". The \"/<soft-key>\" and \"//<hard-key>\" may be repeated and mixed`. The \"///password\" is optional and should only occur once.",
   "You don't have any accounts. Some features are currently hidden and will only become available once you have accounts.": "You don't have any accounts. Some features are currently hidden and will only become available once you have accounts.",
+  "You have {{claimCount}} accounts that need attestations. Use the Claim Tokens app in the navigation bar to complete the process. Until you do, your balances for those accounts will not be reflected.": "You have {{claimCount}} accounts that need attestations. Use the Claim Tokens app in the navigation bar to complete the process. Until you do, your balances for those accounts will not be reflected.",
```

Comment:
Not sure why this is the same as the line below - seems we have a fix applied that may not reflect everywhere. ... yes, it sucks.

Reply:
Adjusted in https://github.com/polkadot-js/apps/pull/3829